NATIVE_EARN.md

```diff
@@ -0,0 +1,58 @@
+# Create Native Earn Offer
+In order to create a native earn offer in your app
+1. Generate a JWT that represent a EarnOffer signed by you. (see [spend offer jwt specs](#earn-offer-jwt-specs))
+2. Call the method: `Kin.requestPayment(offerJwt, callback);` (see [example below](#example-from-sample-app))
```

Comment:
3. The Ecosystem backend service will validate offerJwt and will create a blockchain transactions on behalf of the digital service to the specified user. Ecosystem Service backend should hold sufficient Kin funds for this offer.  While testing Ecosystem backend service will accept any payment request. On real use case digital service need to have an open account with sufficient funds - please contact us for more details.